ethcore/src/spec/spec.rs

```diff
@@ -194,6 +194,7 @@ impl CommonParams {
 			let mut wasm = ::vm::WasmCosts::default();
 			if block_number >= self.kip4_transition {
```

Comment:
should be `kip6_transition`, right?

we've created separate kip for that

Reply:
it’s approved? 🙂

Reply:
if it will get updated, we'll update the code

it won't be in effect until chainspec changed